### PR TITLE
Tftp container: Use localhost and http by default

### PR DESCRIPTION
**Pull request by aaannz**
## What does this PR change?

To prevent hairpin problems and also because we are by default running in pod with other containers, we want to connect to the localhost instead of FQDN.
With localhost we use http.

PR includes black linting of the file.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/23966

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [x] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by aaannz** [comment]
For review check per-commit. Last commit is black linting.

**Comment by m-czernek** [comment]
Re failing linting, taking a look - the flow should not fail outside of the three directories specified in the flow.

**Comment by m-czernek** [comment]
If you'd like Ondra, you can include commits from https://github.com/uyuni-project/uyuni/pull/8426 in your PR to fix the failing check.

Otherwise, you can wait for it to be merged and then rebase.

File: containers/proxy-tftpd-image/tftp_wrapper.py [hunk 1]
```diff
@@ -82,158 +97,229 @@ def contentFilter(self):
...
+
 def main():
     args = get_arguments()
     logging.getLogger().setLevel(logging.getLevelName(args.logLevel.upper()))
 
     try:
         with open(args.configFile) as source:
             config = yaml.safe_load(source)
-            args.httpHost = config['proxy_fqdn']
+            args.proxyFqdn = config.get("proxy_fqdn", "localhost")
```
Comment (cbosdo) [inline containers/proxy-tftpd-image/tftp_wrapper.py:294]: Do we get the `proxy_fqdn` config from the container config?

Comment (aaannz) [reply containers/proxy-tftpd-image/tftp_wrapper.py:294]: Yes, in our deployment it's always set correctly from the config file. But I see a mistake here. As a `get` default there should be `args.proxyFqdn`.
This is more for standalone usage without using any config file.

**Review by cbosdo — APPROVED** [review]
LGTM. The failing Java unit tests are the 2 usual flaky ones.

**Review by agraul — APPROVED** [review]

**Pull request merged**